src/main/java/net/dv8tion/jda/api/events/guild/voice/GenericGuildVoiceEvent.java

```diff
@@ -33,6 +33,10 @@
  *
  * <p>These events require the {@link net.dv8tion.jda.api.utils.cache.CacheFlag#VOICE_STATE VOICE_STATE} CacheFlag to be enabled, which requires
  * the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_VOICE_STATES GUILD_VOICE_STATES} intent.
+ *
+ * <br>{@link net.dv8tion.jda.api.JDABuilder#createLight(String) createLight(String)} disables that CacheFlag by default!
+ *
+ * <p>The {@link net.dv8tion.jda.api.entities.Member Member} must be cached for these events to fire for the Member in question.
```

Comment:
I would prefer these to look more like this https://github.com/DV8FromTheWorld/JDA/blob/4f81b76a2bc2a715c9c3af1391ec1708e30dfb36/src/main/java/net/dv8tion/jda/api/events/user/UserActivityStartEvent.java#L39-L51 for better consistency and clarity.

Reply:
Got it